pipelines/operator.py

```diff
@@ -87,27 +91,40 @@ def _create_seldon_logger(self):
         logger = LOGGER.substitute({
             'experimentId': self._experiment_id
         })
-
         return logger
 
+    def _get_dataset_from_parameters(self):
+        dataset = None
+        if self._parameters:
+            for parameter in self._parameters:
+                parameter_name = parameter.get('name')
+                if parameter_name == 'dataset':
+                    dataset = parameter.get('value')
+        return dataset
+
     def create_container_op(self):
         """Create operator operator from YAML file."""
+        arguments = []
+        if 'datasets' not in self._image:
+            commands = ['sh', '-c']
```

Comment:
Acho que o ideal é não ter este `if   if 'datasets' not in self._image:` .ainda fica um pouco cravado no código, né....
É muito trabalho colocar o "entrypoint" como campo lá da task tbm, @shiguemi32 ?

Reply:
Nem, pode deixa que eu coloco